.github/workflows/deployment-test.yml

```diff
@@ -11,13 +11,10 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - run: echo "::set-output name=version::$(cat .nvmrc)"
-        id: nvmrc
-
       - name: Setup node
         uses: actions/setup-node@v3
         with:
-          node-version: "${{ steps.nvmrc.outputs.version }}"
+          node-version-file: ".nvmrc"
           cache: "npm"
```

Comment:
Maybe we should change this to `yarn` as well, since that's more correct & it will make it consistent? 🤔

```suggestion
          cache: "yarn"
```

Reply:
@MichaelDeBoey These ones may use `npm` cache since they are being used with `scripts/deployment-test` as the working directory. Inside the `scripts/deployment-test` we have a `package-lock.json` file instead of a `yarn.lock` one.

Reply:
Ok, so should we remove the `package-lock.json` inside `scripts/deployment-test` and update the `.gitignore` accordingly?

Reply:
that'd be :chefkiss: - skipped that when i tweaked that set up recently

Reply:
Since all commands spawn inside the deployment test scripts use `npm`, I suggest keeping installing the dependencies inside the `scripts/deployment-test` folder. @mcansh What do you think?

Reply:
I think it would be better to be consistent tbh as it doesn't matter if scripts are using `npm`/`yarn` for the cached version

Reply:
@MichaelDeBoey @mcansh updated all to use Yarn!